frontend/components/AccountDeleted.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
```

Comment:
Consider renaming your JSX files to have the `.jsx` extension. [See relevant discussion](https://github.com/airbnb/javascript/pull/985)

Reply:
It seems that `.js` is the norm when using Next.js, even when pages and components include `JSX`.